tests/x11/pidgin/pidgin_IRC.pm

```diff
@@ -57,14 +60,23 @@ sub run {
     send_key "alt-j";
 
     # Should open sledtesting channel
-    assert_screen 'pidgin-irc-sledtesting';
+    # we need to switch tabs on SLE15
```

Comment:
This could be also just `assert_and_click 'pidgin-irc-sledtesting';`
no need to change it, just idea :thinking:

Reply:
Not sure about the pre-SLE15 behavior. I am not sure, what the click can do.

Reply:
no problem :+1: